tests/core/server/test_dos.py

```diff
@@ -244,6 +244,47 @@ def is_banned():
 
         await time_out_assert(15, is_banned)
 
+    @pytest.mark.asyncio
+    async def test_peer_gossip_not_rate_limited(self, setup_two_nodes):
+        """
+        We trigger the outgoing rate limit, then test that peer gossip messages
+        are let through as a special case.
```

Comment:
we have to rate limit our outgoing peer gossip messages. Otherwise the other end will disconnect us!

Reply:
ok - so that means we need to do a bit more than what we are going in chia/server/ws_connection.py:360